samples/cpp/speech_sample/main.cpp

```diff
@@ -245,6 +245,7 @@ int main(int argc, char* argv[]) {
         gnaPluginConfig[ov::intel_gna::compile_target.name()] = parse_target(FLAGS_compile_target);
         gnaPluginConfig[ov::intel_gna::memory_reuse.name()] = !FLAGS_memory_reuse_off;
         gnaPluginConfig[ov::intel_gna::pwl_max_error_percent.name()] = FLAGS_pwl_me;
+        gnaPluginConfig[ov::log::level.name()] = FLAGS_log;
```

Comment:
There is no equivalence btw C++ and Python samples.
I see python `speech_sample` is much more simplified.
IMO, If really needed it could be added separately.
Beside, I think Python `speech_sample` should be renamed in future to avoid confusion.

Reply:
Resolved offline